megamek/src/megamek/common/DockingCollar.java

```diff
@@ -67,16 +68,15 @@ protected DockingCollar() {
     // Public constructors and methods.
 
     /**
-     * Create a space for the given tonnage of troops. For this class, only the
-     * weight of the troops (and their equipment) are considered; if you'd like
-     * to think that they are stacked like lumber, be my guest.
+     * Create a Jumpship collar that can carry one dropship
      *
-     * @param docks
-     *            - The weight of troops (in tons) this space can carry.
+     * @param docks - Capacity. A collar can always carry one dropship.
+     * @param collarNumber - the Id of this collar, used for tracking in MHQ 
```

Comment:
The dash is unnecessary after the parameter name.

Reply:
Fixed